lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -5112,6 +5138,8 @@ LogicalResult StmtEmitter::visitSV(BindOp op) {
 
 LogicalResult StmtEmitter::visitSV(InterfaceOp op) {
   emitComment(op.getCommentAttr());
+  SmallVector<std::pair<std::string, std::string>> waiverStrs;
+  emitWaivers(op->getAttrOfType<ArrayAttr>("waivers"), waiverStrs);
```

Comment:
Can we tweak a helper function a bit for simplisity?
```suggestion
void EmitterBase::emitWaivers(
    const SmallVector<std::pair<std::string, std::string>> &waiverStrs,
    bool moduleEnd) {
 ...
  auto waiverStrs = hw::getWaivers(op);
  emitWaivers(waiverStrs, /*moduleEnd=*/false);
  ...
  emitWaivers(waiverStrs, /*moduleEnd=*/true);
```